### PR TITLE
Get user tests

### PR DESCRIPTION
**Pull request by fbalak**
Describe negative and positive get_user test.

### CODE REVIEW

File: api/tendrl/user_get.rst [hunk 1]
```diff
@@ -1,2 +1,44 @@
 user_get
```
Comment (mkudlej) [inline api/tendrl/user_get.rst]: Please change test name

File: api/tendrl/user_get_negative.rst [hunk 2]
```diff
@@ -1,2 +1,29 @@
 user_get_negative
```
Comment (mkudlej) [inline api/tendrl/user_get_negative.rst]: Please change test name

File: api/tendrl/user_get.rst [hunk 3]
```diff
@@ -1,2 +1,44 @@
...
+
+Description
+===========
+
+Check validity of responses from `List Users`_ and `Get Single User`_ API calls.
+
+Setup
+=====
+
+#. Create user via `valid_new_user` fixture populated with user data specified
```
Comment (mkudlej) [inline api/tendrl/user_get.rst]: Please describe what user should be created. I think it is no good to link specific fixtures or code directly.

**Review by mkudlej — CHANGES_REQUESTED** [review]

**Review by mbukatov — APPROVED** [review]

**Review by mkudlej — APPROVED** [review]
LGTM

**Pull request merged**